curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64fb1c4dc0feb219149a7c7d.md

```diff
@@ -16,19 +16,20 @@ Inside the `closeTaskFormBtn` event listener, use `const` to create another vari
 Your `formInputValuesUpdated` variable should check if `titleInput.value` is not equal to `currentTask.title`.
 
 ```js
-assert.match(code, /const\s+formInputValuesUpdated\s*=\s*titleInput\.value\s*!==\s*currentTask\.title\s*/)
+assert.match(code, /const\s+formInputValuesUpdated\s*=\s*(?:(?:titleInput\.value\s*(!==|!=)\s*currentTask\.title));?/)
```

Comment:
Is it better to use `!==?` instead of `*(!==|!=)`?

Reply:
I used to think not, but this is surprisingly a lot more clearer and easier to read. In addition it makes the intention well laid out when the next person goes to update it.